code-studio/scripts/build-commands.js

```diff
@@ -77,13 +77,16 @@ exports.bundle = function (config) {
   };
 
   // Create browserify instance
-  var bundler = browserify({
+  var bundler = browserifyInc({
     // Enables source map
     debug: true,
 
     // Allow requiring jsx without specifying extension
     extensions: ['.jsx'],
 
+    // Required for browserify-incremental
+    cacheFile: shouldWatch ? undefined : outPath(srcPath + commonFile).replace(/\.js?$/i, '.cache.json'),
```

Comment:
Re: browserify-incremental vs. watchify, what happened when I just gave this `cacheFile` all the time was that watchify didn't pick up changes.  I suspect this is because watchify depends on its in-memory cache to know what files it should be watching, and providing the cacheFile argument somehow breaks that.

As it is now, they both work as expected, but neither gets the benefit of the other: Watchify's initial startup will always take the full build time, even if there's a cache file, and non-watch builds only benefit from the most recent non-watch build (ignoring any in-memory caching the watcher did).

I still think this is a strict improvement to the current situation.